topics/css-framework/index.md

```diff
@@ -13,3 +13,5 @@ wikipedia_url: https://en.wikipedia.org/wiki/CSS_framework
 * [Responsiveness](https://github.com/topics/responsive) (often times including breakpoints and [viewport](https://github.com/topics/viewport))
 
 They often have their own class notation and many use [preprocessors](https://github.com/topics/css-preprocessor) like [Sass](https://github.com/topics/sass), [Less](https://github.com/topics/less) or [Stylus](https://github.com/topics/stylus).
+
+There's a collection of [CSS frameworks](https://github.com/collections/css-frameworks).
```

Comment:
I was thinking we should change `There's` to `Here's` but then the link is in the sentence, so I wasn't sure how best to write it. What do you think about using this?

```suggestion
Related Collections:
  - [CSS frameworks](https://github.com/collections/css-frameworks)
```

Reply:
Okay

Reply:
Would be more formal and less boilerplate text 👍🏻